docs/commands/README.md

```diff
@@ -648,11 +648,13 @@ Deploy workers to hosts, described in hosts.yaml
 ```
 USAGE
   $ fluence workers deploy [--no-input] [--relay <value>] [--force] [--timeout <value>] [--ttl <value>] [-k <value>]
-    [--aqua-logs]
+    [--aqua-logs] [--direct] [-k <value>]
 
 FLAGS
   -k, --key-pair-name=<name>  Key pair name
+  -k, --privKey=<value>       !WARNING! for debug purposes only. Passing private keys through flags is unsecure
   --aqua-logs                 Enable Aqua logs
+  --direct                    Deploy workers directly to hosts (use hosts.yaml config instead of deals.yaml)
```

Comment:
I think `fluence workers deploy` should always be `--direct`. Deals should be deployed via `fluence deal create` or `fluence deal deploy`